src/django/pfb_analysis/views.py

```diff
@@ -84,15 +89,12 @@ def perform_create(self, serializer):
                             name=slugify(serializer.validated_data['label']))
 
 
-class NeighborhoodGeoJsonViewSet(ReadOnlyModelViewSet):
+class NeighborhoodGeoJsonViewSet(ReadOnlyModelViewSet, NeighborhoodMixin):
```

Comment:
These base classes should be in the opposite order (also on `NeighborhoodViewSet` above)
```
>>> from pfb_analysis.views import NeighborhoodGeoJsonViewSet
>>> NeighborhoodGeoJsonViewSet.__mro__
(<class 'pfb_analysis.views.NeighborhoodGeoJsonViewSet'>, <class 'rest_framework.viewsets.ReadOnlyModelViewSet'>,
<class 'rest_framework.mixins.RetrieveModelMixin'>, <class 'rest_framework.mixins.ListModelMixin'>,
<class 'rest_framework.viewsets.GenericViewSet'>, <class 'rest_framework.viewsets.ViewSetMixin'>,
<class 'rest_framework.generics.GenericAPIView'>, <class 'pfb_analysis.views.NeighborhoodMixin'>,
<class 'rest_framework.views.APIView'>, <class 'django.views.generic.base.View'>, <type 'object'>)